README.md

```diff
@@ -1,14 +1,142 @@
 3. Run the frontend locally using
 `sbt 'run 9717 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes'`
 
+### End-Points
+#### POST /journey
+
+---
+Creates a new journey, storing the journeyConfig against the journeyId.
+#### Request:
+Request body must contain the continueUrl.
+
+```
+{"continueUrl" : "/testUrl"}
+```
+
+#### Response:
+Status: **Created(201)**
+
+Example Response body:
+
+```
+{“journeyStartUrl” : "/testUrl"}
+```
+
+#### GET /journey/:journeyId
+
+---
+Retrieves all the journey data that is stored against a specific journeyID.
+#### Request:
+A valid journeyId must be sent in the URI
+
+#### Response:
+Status:
+
+| Expected Response                       | Reason
+|-----------------------------------------|------------------------------
+| ```OK(200)```                           |  ```JourneyId exists```
+| ```NOT_FOUND(404)```                    | ```JourneyId doesn't exist```
+
+Example response body:
+```
+{"fullName":
+   {"firstName":"John",
+   "lastName":"Smith"
+   },
+"dateOfBirth": 1978-01-05,
+"nino": "AA111111A",
+"sautr": "1234567890"
+}
+```
+
+### Test End-Points
+#### GET test-only/feature-switches
+
+---
+Shows all feature switches:
+1. Sole Trader Identification Frontend
+
+    - Use stub for Authenticator API
+   
+#### GET/POST test-only/create-journey
+
+---
+This is a test entry point which simulates a service making the initial call to setup a journey.
+
+1. ContinueURL(Required)
+
+   - Where to redirect the user after the journey has been completed
+
+#### GET test-only/retrieve-journey/:journeyId
+
+---
+Retrieves all the journey data that is stored against a specific journeyID.
+
+##### Request:
+A valid journeyId must be sent in the URI
+
+##### Response:
+Status:
+
+| Expected Response                       | Reason
+|-----------------------------------------|------------------------------
+| ```OK(200)```                           |  ```JourneyId exists```
+| ```NOT_FOUND(404)```                    | ```JourneyId doesn't exist```
+
+Example response body:
+```
+{"fullName":
+   {"firstName":"John",
+   "lastName":"Smith"
+   },
+"dateOfBirth": 1978-01-05,
+"nino": "AA111111A",
+"sautr": "1234567890"
+}
+```
+
+#### POST test-only/authenticator/match
+
+---
+
+Stubs the call to Authenticator to check the data submitted
+
+##### Request:
+Request body must contain the full name, date of birth and National Insurance number. SAUTR is optional. 
+
+Example Request body:
+```
+{"fullName":
+   {"firstName":"John",
+   "lastName":"Smith"
+   },
+"dateOfBirth": 1978-01-05,
+"nino": "AA111111A",
+"sautr": "1234567890"
+}
+```
+##### Response:
+Status:
+
+| Expected Response                       | Reason
+|-----------------------------------------|------------------------------
+| ```OK(200)```                           |  ```Match successful```
+| ```UNAUTHORIZED(401)```                 | ```Match not successful```
+| ```FAILED_DEPENDENCY(424)```            | ```Matched with deceased record```
+
```

Comment:
We probably don't need to document the stubs as they're documented by the service we're stubbing.

Can remove this one.